.github/workflows/main.yml

```diff
@@ -77,28 +77,23 @@ jobs:
       - name: Toolchain setup
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2022-01-14
+          toolchain: 1.60.0
           override: true
           profile: minimal
           components: llvm-tools-preview
       - name: Cache youki
         uses: Swatinem/rust-cache@v1
-      - name: install cargo-llvm-cov
-        env:
-          CARGO_LLVM_COV_VERSION: 0.1.5
-        run: |
-          wget https://github.com/taiki-e/cargo-llvm-cov/releases/download/v${CARGO_LLVM_COV_VERSION}/cargo-llvm-cov-x86_64-unknown-linux-gnu.tar.gz -qO- | tar -xzvf -
-          mv cargo-llvm-cov ~/.cargo/bin
       - name: Update System Libraries
         run: sudo apt-get -y update
       - name: Install System Libraries
         run: sudo apt-get install -y pkg-config libsystemd-dev libdbus-glib-1-dev libelf-dev libseccomp-dev
+      - name: Install grcov
+        run: cargo install grcov
       - name: Run Test Coverage for youki
         run: |
-          cd ./crates
-          cargo llvm-cov clean --workspace
-          cargo llvm-cov --no-report
-          cargo llvm-cov --no-run --lcov --ignore-filename-regex "libcgroups/src/systemd/dbus/systemd_api.rs" --output-path ./coverage.lcov
+          cargo clean
+          LLVM_PROFILE_FILE="coverage.profraw" RUSTFLAGS="-Cinstrument-coverage" cargo build 
+          grcov . --binary-path ./target/debug -s . -t lcov --branch --ignore-not-existing -o ./coverage.lcov
```

Comment:
grcov itself does not have the ability to run tests, so the new commands actually just build the code. As a result, coverage is reported as 0 percent.

Reply:
Oops! I was following something, and forgot that we need to manually run tests :sweat_smile: :sweat_smile: 
Thanks a lot!